codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AwsHttpPresignURLClientGenerator.java

```diff
@@ -21,6 +21,7 @@
 import java.util.Set;
 import java.util.TreeMap;
 import java.util.TreeSet;
+import javax.swing.plaf.synth.SynthMenuItemUI;
```

Comment:
Is this unused? You probably didn't want to pull this in regardless.

Reply:
Thanks, not quire sure where that came from, but yeah should not be added!